packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -729,8 +728,7 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
     if (copySampleCountsAndTags) {
       copy
         ..exclusiveSampleCount = exclusiveSampleCount
-        ..inclusiveSampleCount =
-            resetInclusiveSampleCount ? null : inclusiveSampleCount;
```

Comment:
removing this makes me a bit nervous. Anywhere `shallowCopy` was used where `resetInclusiveSampleCount: false` was not specified will now be affected. For example, `CpuStackFrame.shallowCopy` is used in the filtering code (filtering by user tag or source url). I'd be a bit nervous that these code paths are affected by this change. Have you tried filtering by a user tag or by the filter categories (hide native, hide flutter, etc.)?

Reply:
I just checked, I don't see any regressions with this patch vs the current release.